cluster-operator/src/main/java/io/strimzi/operator/cluster/model/CruiseControlConfiguration.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.operator.cluster.model;
+
+import io.strimzi.api.kafka.model.CruiseControlSpec;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+
+/**
+ * Class for handling Cruise Control configuration passed by the user
+ */
+public class CruiseControlConfiguration extends AbstractConfiguration {
+
+    /**
+     *  The list of goals that will be used by default if no goal list is provided. This list of goal
+     *  will also be used for proposal pre-computation. If default.goals is not specified, it will
+     *  be default to goals config.
+     */
+    private static final String DEFAULT_GOALS = "com.linkedin.kafka.cruisecontrol.analyzer.goals.RackAwareGoal," +
+        "com.linkedin.kafka.cruisecontrol.analyzer.goals.ReplicaCapacityGoal," +
```

Comment:
Are goals always given as fully qualified class names, or can they use the unqualified name as an alias?

Reply:
They are FQCN in the CC configuration file. I suppose using this does mean you could use your own Goal at some point provided you add the jar to the CC image?

Reply:
@tomncooper Yes